.goreleaser.yml

```diff
@@ -0,0 +1,104 @@
+before:
+  hooks:
+    - go mod tidy
+    - go test ./...
+builds:
+  - id: fw-id-agent
+    binary: fw-id-agent
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+    main: ./cmd/fw-id-agent/main.go
+    ldflags:
+      - -s -w -X github.com/T-Systems-MMS/fw-id-agent/internal/agent.Version={{.Version}}-{{.Commit}}
+  - id: fw-id-cli
+    binary: fw-id-cli
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+    main: ./cmd/fw-id-cli/main.go
+    ldflags:
+      - -s -w -X github.com/T-Systems-MMS/fw-id-agent/internal/agent.Version={{.Version}}-{{.Commit}}
+archives:
+  - format: tar.gz
+    rlcp: true
+    wrap_in_directory: true
+    files:
+      - src: init/fw-id-agent.service
+        dst: fw-id-agent.service
+        info:
+          mode: 0644
+      - src: configs/config.json
+        dst: example_config.json
+        info:
+          mode: 0644
+      - README.md
+      - LICENSE
+checksum:
+  name_template: "checksums.txt"
+snapshot:
+  name_template: "{{ incpatch .Version }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+      - "^docs:"
+      - "^test:"
+nfpms:
+  - package_name: fw-id-agent
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Arch }}"
+    vendor: Deutsche Telekom MMS GmbH
+    maintainer: The MMS Linux Dev Team <mmslinux-dev@t-systems-mms.com>
+    description: |-
+      Corporate Identity Agent
+      Agent to logon at the corporate firewall.
+    license: MIT
+    builds:
+      - "fw-id-agent"
+      - "fw-id-cli"
+    formats:
+      - deb
+    bindir: /usr/bin
+    section: net
+    # important: this makes the package non native
+    release: "1"
+    priority: important
+    # chglog init -> generates changelog.yml
+    # chglog add --version v#.#.# -> after every tag, call in hook above?
+    # activate if https://github.com/goreleaser/nfpm/pull/656 is merged and used in goreleaser
+    # -> remove overrides
+    # changelog: "changelog.yml"
+    scripts:
+      postinstall: build/package/postinstall.sh
+      preremove: build/package/preremove.sh
+      postremove: build/package/postremove.sh
+    deb:
+      lintian_overrides:
+        - "no-manual-page"
+        - "no-changelog"
+        - "statically-linked-binary"
+        - "maintainer-script-calls-systemctl"
+    dependencies:
+      - "libc6 (>= 2.34)"
```

Comment:
Not sure we need this with the statically linked executables. Is this a debian packaging thing?

Reply:
I removed it after i checked the binary with ldd.